helm/values.yaml.liquid

```diff
@@ -0,0 +1,4 @@
+secrets:
+    deployToken: {{ configuration.deployToken }}
```

Comment:
Not sure if this will have any impact, but indentation is 4 spaces instead of 2.